docs/StyleGuide/Style-Guide.md

```diff
@@ -18,10 +18,17 @@ Any **lemmas**, **predicates**, **functions**, **methods**, **classes**, **modul
 are named with `PascalCase`.
 ```dafny
 method FindIndex(arr: seq<int>, k: int)
-    ...
+  ...
 ```
 
-Any static or global **constants** are named with `UPPERCASE_WITH_UNDERSCORES`.
+The **lemma** keyword indicates a ghost method used for proof purposes. Any **lemma** names should be prefixed with `Lemma`.
```

Comment:
FWIW, I don't follow this convention.  We don't prefix method names with `Method` and function names with `Function`, and `lemma LemmaXYZ` looks very redundant.

Reply:
I would agree; I was not editing the documents, just merging them.

Reply:
I removed it, to fit with the philosophy of not including redundant information in names.